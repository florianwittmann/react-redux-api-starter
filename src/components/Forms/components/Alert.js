import React from "react";

export default props => {
  if (!props.errorMessage) return null;

  return (
    <div className="notification is-danger">
      <strong>Ooops!</strong> {props.errorMessage}
    </div>
  );
};
