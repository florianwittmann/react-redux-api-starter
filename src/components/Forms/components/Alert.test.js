import React from "react";
import { shallow } from "enzyme";
import Alert from "./Alert";

test("should render empty alert without errorMessage", () => {
  const wrapper = shallow(<Alert />);
  expect(wrapper).toMatchSnapshot();
});

test("should render alert with correct errorMessage", () => {
  const wrapper = shallow(<Alert errorMessage="Some example error message." />);
  expect(wrapper).toMatchSnapshot();
});
