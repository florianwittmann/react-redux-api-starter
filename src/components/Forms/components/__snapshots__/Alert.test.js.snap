// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render alert with correct errorMessage 1`] = `
<div
  className="notification is-danger"
>
  <strong>
    Ooops!
  </strong>
   
  Some example error message.
</div>
`;

exports[`should render empty alert without errorMessage 1`] = `""`;
